src/externalConfigFileHandler.ts

```diff
@@ -0,0 +1,71 @@
+import path from "path"
+import fs from "fs"
+import bytes from "bytes"
+// eslint-disable-next-line import/no-internal-modules
+import compressedSize from "bundlesize/src/compressed-size"
+
+import { BundleSizeConfig } from "./index"
+
+export const createNewConfigFile = (
+  oldConfig: BundleSizeConfig,
+  delta: number,
+  targetSize: number,
+  buildDir: string,
+  fileName?: string
+) => {
+  if (!fileName) {
+    return
+  }
+
+  try {
+    const newConfig = updateConfigurationWithNewBundleSizes(
+      oldConfig,
+      delta,
+      targetSize
+    )
+    fs.writeFileSync(
+      path.join(buildDir, "new-" + fileName),
+      JSON.stringify(newConfig)
+    )
+  } catch (error) {
+    // eslint-disable-next-line no-console
+    console.log("Cannot create new config file", error)
```

Comment:
I'd print to error messages SDTERR:
```suggestion
    console.error("Cannot create new config file", error)
```

Reply:
For cmd lines, I prefer to just throw the error. there is little additional information this manual handling gives us plus it just moves on in the code, even though something severe happened

Reply:
I don't want to fail the pipeline here. It's fine in my opinion and not a blocker for the pipeline when it fails here. That is why I only log it